### ISSUE TITLE
Main Anysoft install is not in FDroid app

### ISSUE THREAD

**Issue by francesmothrock** [AnySoftKeyboard/AnySoftKeyboard#2211]
When I search for AnySoft in FDroid, I get a list of language packs to install, but can't find the primary app. It's extra weird because I can find it on the FDroid web site on the computer. 

This is a new phone I'm setting up and haven't installed AnySoft on it before.

Android Version 10
Google Pixel 3a

**Comment by todduk** [comment]
Don't bother, it's a waste of your time.

**Comment by somini** [comment]
For some reason, the name on F-Droid is now "Sinhala for AnySoftKeyboard". It's the same app ID, but the name is different.

**Comment by somini** [comment]
See also #2161, this sounds like a bug on the build process.

https://f-droid.org/packages/com.menny.android.anysoftkeyboard/

**Comment by somini** [comment]
See also #2103

**Comment by francesmothrock** [comment]
Thanks Somini. I had seen other posts where it was merged or mis-named with various other languages. But for me it showed up as Sinhala for AnySoftKeyboard, just like you said. Glad to know a fix is in the works.
Cheers.

**Comment by realmain** [comment]
Has there been any update on this? It's July and the issue still persists.

**Comment by ilmaisin** [comment]
This is also being discussed in F-Droid bug tracker: https://gitlab.com/fdroid/fdroiddata/-/issues/2014

IMO this issue needs much more attention. This is a real show-stopper particularly for non-technical users. Please at least close all the duplicate issues (at least #2103, #2161, #2211 and #2469) and start the attempts to troubleshoot this.

**Comment by ngrigoriev** [comment]
Also the version shown for https://f-droid.org/packages/com.menny.android.anysoftkeyboard/ is "Version 1.10.1109 (6279)  Added on 2020-02-12". At the same time, Play store has 1.10.1794 updaed on Mar 21, 2021. Quite a gap.

**Comment by ilmaisin** [comment]
Yes, ASK's F-Droid release seems to be abandonded to its own devices. It probably should be taken down and moved to the archive repo.

31. maaliskuuta 2021 21.46.53 GMT+03:00 Nikolai Grigoriev ***@***.***> kirjoitti:
